javascript/javascript3/week3/shopping.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <select id="from_currency">
```

Comment:
The convert to currency method seems to work, but this part in HTML is not really doing anything, so not sure why it is here. It is not submitting and not calling the function, so should be either edited to do that or removed completely. 
The function itself is good though, good job! 💪🏽